frontend/src/routes/Infrastructure/Clusters/ManagedClusters/components/DistributionField.tsx

```diff
@@ -290,6 +290,47 @@ export function DistributionField(props: {
                 />
             </>
         )
+    } else if (props.cluster.distribution.upgradeInfo?.posthookDidNotRun) {
+        // CURATOR POSTHOOK JOB DID NOT RUN
+        return (
+            <>
+                <div>{props.cluster?.distribution.displayVersion}</div>
+                <AcmInlineStatus
+                    type={StatusType.danger}
+                    status={t('upgrade.upgradefailed', {
+                        version: props.cluster?.consoleURL
+                            ? ''
+                            : props.cluster?.distribution.upgradeInfo.desiredVersion,
+                    })}
```

Comment:
Translation key `upgrade.upgradefailed` has no interpolation variable called `version` and it seems odd that you would set this to an empty string if it did.